packages/stack/blockchain/src/index.js

```diff
@@ -38,8 +40,20 @@ export default class Blockchain {
       if (!stopFn) {
         throw new Error(`Blockchain client '${clientName}' must be registered with a 'stopFn' function, client not registered.`);
       }
+      if (!provider) {
+        // Set default provider function
+        clientFunctions.provider = async () => {
+          if (this.blockchainConfig.endpoint.startsWith('ws')) {
+            return new Web3.providers.WebsocketProvider(this.blockchainConfig.endpoint, {
+              headers: { Origin: constants.embarkResourceOrigin }
```

Comment:
Should we add
```
clientConfig: {
  fragmentationThreshold: 81920
}
```
Or are we confident this has been resolved with Geth 1.9?

Reply:
Yes, there was a fix in 1.9 and we now only support 1.9+. I'll test again to be sure, but it should be good to go

Reply:
Just tested Teller again with geth and it worked! It was one of the projects that didn't with 1.8